docs/guides/serverless-port-code.mdx

```diff
@@ -17,6 +17,9 @@ This documentation is relevant to the new IBM Quantum&reg; Platform. If you need
 </CloudContent>
 
 The following example demonstrates how to port existing code to leverage Qiskit Serverless.
+<Admonition type="note">
+The following code assumes that you have saved your credentials. If you have not, follow the instructions in <LegacyContent>[Set up an IBM Quantum channel](/guides/setup-channel#iqp)</LegacyContent><CloudContent>[Set up your IBM Cloud account](/guides/cloud-setup#cloud-untrusted)</CloudContent> to authenticate with your API token.
```

Comment:
```suggestion
The following code assumes that you have saved your credentials. If you have not, follow the instructions in <LegacyContent>[Set up an IBM Quantum channel](/guides/setup-channel#iqp)</LegacyContent><CloudContent>[Set up your IBM Cloud account](/guides/cloud-setup)</CloudContent> to authenticate with your API key.
```

Reply:
It seems helpful to point to the correct step, instead of just linking to the top of the page and making them dig for it.